### PR TITLE
Added Include plugin to readme

### LINKED ISSUE #52: Add figma-include-accessibility-annotations Plugin
**Is your resource open source?**
Yes

**Is your request for a plugin, widget, or developer resource?**
Plugin

**What is the URL for your resource repo?**
[figma-include-accessibility-annotations](https://github.com/eBay/figma-include-accessibility-annotations)

### PR DESCRIPTION
**Pull request by mkbhuiyan96**
This resolves #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,15 @@ A collection of plugins that have been open-sourced. You can install these plugi
 
 ### Accessibility
 
+#### Include
+[SOURCE CODE](https://github.com/eBay/figma-include-accessibility-annotations) · [PLUGIN](https://www.figma.com/community/plugin/1208180794570801545/includeaccessibility-annotations) · [APACHE LICENSE 2.0](https://github.com/eBay/figma-include-accessibility-annotations/blob/main/LICENSE)
+
+Include is a Figma plugin that simplifies annotating for accessibility (a11y). It makes it easier for designers to spec and easier for developers to understand what is required.
+
+**[⬆ Back to TOC](#table-of-contents)**
```
Comment (akbarbmirza) [inline README.md:159]: This bookmark back to TOC doesn't belong in this commit. There are a few more entries in this section. There is a bookmark at the end of the section. Please remove this and the extra newline.

**Review by akbarbmirza — CHANGES_REQUESTED** [review]
Add the missing newline and remove the Back to TOC bookmark and this is ready to be merged.

**Review by akbarbmirza — APPROVED** [review]

**Comment by akbarbmirza** [comment]
hey @mkbhuiyan96 do you have a LinkedIn and Twitter? Would love to shout you out!

**Comment by mkbhuiyan96** [comment]
Hi @akbarbmirza I don't have Twitter but here's [my LinkedIn.](https://www.linkedin.com/in/mohammad-bhuiyan-a5aa91242/) Thanks!

**Pull request merged**